hetmech/test_matrix.py

```diff
@@ -0,0 +1,51 @@
+import numpy
+from scipy import sparse
+import hetio.readwrite
+import hetio.hetnet
+
+from .matrix import get_node_to_position, metaedge_to_adjacency_matrix
+
+
+def test_sparse_matrix():
+    '''
+    Test the functionality of metaedge_to_adjacency_matrix in
+    generating sparse matrices vs numpy arrays. Uses same test
+    data as in test_degree_weight.py Figure 2D of Himmelstein &
+    Baranzini (2015) PLOS Comp Bio.
+    https://doi.org/10.1371/journal.pcbi.1004259.g002
+    '''
+
+    url = 'https://github.com/dhimmel/hetio/raw/{}/{}'.format(
+        '9dc747b8fc4e23ef3437829ffde4d047f2e1bdde',
+        'test/data/disease-gene-example-graph.json',
+    )
+    graph = hetio.readwrite.read_graph(url)
+
+    def inef_metaedge_to_adjacency_matrix(graph, metaedge, dtype=numpy.bool_):
```

Comment:
You want to remove this complex logic from the test. For this initial PR, let's just try to add tests for the existing `metaedge_to_adjacency_matrix` function without adding any new functionality. In other words, you'll manually construct the adjecency matrices corresponding to the example graph. You'll hardcode these matrices.

Note that tests are different than other types of coding. In most coding, you want to use logic and avoid hardcoding. In tests, you want to hardcode the expected result and check that the function output matches.